src/test/java/com/redhat/rhjmc/containerjfr/commands/internal/ScanTargetsCommandTest.java

```diff
@@ -139,9 +139,19 @@ void shouldExecuteAndReturnExceptionOutputIfPlatformClientThrows() throws Except
     }
 
     List<ServiceRef> getMockServices() {
-        ServiceRef mockA = new ServiceRef("aHost", "Host A", 1);
-        ServiceRef mockB = new ServiceRef("bHost", "Host B", 2);
-        ServiceRef mockC = new ServiceRef("cHost", "Host C", 3);
+        ServiceRef mockA;
+        ServiceRef mockB;
+        ServiceRef mockC;
+        try {
+            mockA = new ServiceRef("aHost", 0, "Host A");
+            mockB = new ServiceRef("bHost", 0, "Host B");
+            mockC = new ServiceRef("cHost", 0, "Host C");
+        } catch (Exception e) {
```

Comment:
This exception should never actually be thrown though, right? I think we can skip trying to handle it here and just add a throws-declaration to this test utility method.